test/DurableTaskEndToEndTests.cs

```diff
@@ -343,5 +346,94 @@ public DurableTaskEndToEndTests(ITestOutputHelper output)
+
+            using (JobHost host = TestHelpers.GetJobHost(nameof(StartOrchestration_OnUnregisteredOrchestrator)))
+            {
+                await host.StartAsync();
+
+                Exception ex = await Assert.ThrowsAsync<FunctionInvocationException>(async () => await host.StartFunctionAsync("UnregisteredOrchestrator", "Unregistered", this.output));
+                
+                Assert.NotNull(ex.InnerException);
+                Assert.Contains(errorMessage, ex.InnerException?.ToString());
+
+                await host.StopAsync();
+            }
+        }
+
+        /// <summary>
+        /// End-to-end test which runs a orchestrator function that calls a non-existent activity function.
+        /// </summary>
+        [Fact]
+        public async Task Orchestration_OnUnregisteredActivity()
+        {
+            const string functionName = "UnregisteredActivity";
+            string errorMessage = $"The function '{functionName}' doesn't exist, is disabled, or is not an activity or orchestrator function";
+
+            using (JobHost host = TestHelpers.GetJobHost(nameof(Orchestration_OnUnregisteredActivity)))
+            {
+                await host.StartAsync();
+
+                var startArgs = new StartOrchestrationArgs
+                {
+                    FunctionName = functionName,
+                    Input = new {Foo = "Bar"}
+                };
+
+                var client = await host.StartFunctionAsync(nameof(TestOrchestrations.CallActivity), startArgs, this.output);
+                var status = await client.WaitForCompletionAsync(TimeSpan.FromSeconds(30), this.output);
+
+                Assert.Equal("Failed", status?.RuntimeStatus);
+
+                // There aren't any exception details in the output: https://github.com/Azure/azure-webjobs-sdk-script-pr/issues/36
+                Assert.True(status?.Output.ToString().Contains(errorMessage));
+
+                await host.StopAsync();
+            }
+        }
+
+        /// <summary>
+        /// End-to-end test which runs a orchestrator function that calls another orchestrator function.
+        /// </summary>
+        [Fact]
+        public async Task Orchestration_OnValidOrchestrator()
+        {
+            const string greetingName = "ValidOrchestrator";
+            const string validOrchestratorName = "SayHelloInline";
+            var input = new {Foo = greetingName};
+            var inputJson = JsonConvert.SerializeObject(input);
+
+            using (JobHost host = TestHelpers.GetJobHost(nameof(Orchestration_OnValidOrchestrator)))
+            {
+                await host.StartAsync();
+
+                var startArgs = new StartOrchestrationArgs
+                {
+                    FunctionName = nameof(TestOrchestrations.SayHelloInline),
+                    Input = input
+                };
+
+                var client = await host.StartFunctionAsync(nameof(TestOrchestrations.CallActivity), startArgs, this.output);
```

Comment:
>CallActivity [](start = 85, length = 12)

Let's add a comment which shows the call chain. Something like CallActivity --> CallActivity --> SayHelloInline.